streamz/core.py

```diff
@@ -319,18 +339,35 @@ def remove_stream(change):
 
         return output._ipython_display_(**kwargs)
 
-    def _emit(self, x):
+    def _emit(self, x, metadata=None):
+        if metadata is None:
+            metadata = {}
+
+        refs = []
+        if 'refs' in metadata:
+            refs = metadata['refs']
+            if not isinstance(refs, list):
```

Comment:
Explain in a comment, when is refs a list versus a single instance?

Reply:
Cleaned up. Refs is always a single instance